src/runtime/builtin_modules/sys.cpp

```diff
@@ -17,6 +17,7 @@
 #include <cmath>
 #include <langinfo.h>
 #include <sstream>
+#include <gmp.h>
```

Comment:
Hardcode the `sys.long_info` is OK, but I think in `sys` more test cases will be passed if it depends on sys.long_info (maybe there is no more test cases depends on it LOL, so I don't know which choice is suitable).